app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  belongs_to :role
```

Comment:
Why `User` belongs to `Role` but has many roles?

Reply:
Because every user have `one role` such as: student, teacher and superadmin, based on the belongs_to, I can make sure that the role_id have to exist in Roles table. Am I wrong?